apis/hive/v1/clusterdeployment_types.go

```diff
@@ -142,6 +142,10 @@ type ClusterDeploymentSpec struct {
 	// provisioning worker machines.
 	// +optional
 	MachineManagement *MachineManagement `json:"machineManagement,omitempty"`
+
+	// BoundServiceAccountSignkingKeySecretRef refers to a Secret that contains a 'bound-service-account-signing-key.key' data key pointing to the private key that will be used to sign ServiceAccount objects. Primarily used to provision AWS clusters to use Amazon's Security Token Service.
```

Comment:
super nit: can you wordwrap the lines at around 80-90, like the other comments. this is currently single line..